lib/daru/dataframe.rb

```diff
@@ -237,7 +237,7 @@ def from_vector_rows source, opts
     def initialize source, opts={} # rubocop:disable Metrics/MethodLength
       vectors, index = opts[:order], opts[:index] # FIXME: just keyword arges after Ruby 2.1
       @data = []
-      @name = opts[:name] || SecureRandom.uuid
+      @name = opts[:name]
```

Comment:
Please put this in a different PR. It needs to show up in the commit history as a separate feature.